app/src/main/java/com/spascoding/taskycourse/feature_agenda/presentation/utils/MonthExtractor.kt

```diff
@@ -0,0 +1,9 @@
+package com.spascoding.taskycourse.feature_agenda.presentation.utils
+
+class MonthExtractor(private val date: String) {
+
+    fun extract(): Int {
+        return date.split("-")[1].toInt() - 1 //month is zero based
+    }
+
+}
```

Comment:
Assuming this is for parsing the date strings from the API response, this is overcomplicated. The format the API responds with is an ISO standard which you can easily parse into LocalDateTime or ZonedDateTime objects:
```
val localDateTime = LocalDateTime.ofInstant(
        Instant.parse(dateString),
        ZoneId.systemDefault()
    )
```
This parses the date string into a LocalDateTime object and also automatically converts it from UTC time zone to the user's local time zone.